config/tracing.go

```diff
@@ -0,0 +1,46 @@
+package config
+
+import (
+	"errors"
+)
+
+type TracingConfig struct {
+	GlobalAttributes        map[string]string    `yaml:"global_attributes"`
+	Opentelemetry           *OpenTelemetryConfig `yaml:"opentelemetry"`
+	ServiceName             string               `yaml:"service_name" default:"WebhookX"`
+	CapturedRequestHeaders  []string             `yaml:"captured_request_headers"`
+	CapturedResponseHeaders []string             `yaml:"captured_response_headers"`
+	SafeQueryParams         []string             `yaml:"safe_query_params"`
+	SamplingRate            float64              `yaml:"sampling_rate" default:"1"`
+}
```

Comment:
How to specific the propagation type?

Reply:
`otel.SetTextMapPropagator(autoprop.NewTextMapPropagator())` will config propagation type from env `OTEL_PROPAGATORS`, default is `tracecontext,baggage`